cmd/backup-manager/app/backup/backup.go

```diff
@@ -211,3 +259,75 @@ func archiveBackupData(backupDir, destFile string) error {
 	}
 	return nil
 }
+
+// getBRCommitTs get backup position from the log output of BR
+// It really depends on the format in BR log,
+// currently, it's in the format of [BackupTS=412992336769581057]
+func getBRCommitTs(out string) (string, error) {
```

Comment:
Please don't get such important info from logs, it's fragile that the log format is not standard and may change in the future.

Reply:
fixed